src/commands/api-mesh/__tests__/log-get-bulk.test.js

```diff
@@ -548,14 +358,11 @@ describe('parsePastDuration', () => {
 		},
 	);
 
-	const invalidDurations = ['20h', '20 hours', '20s', '20 seconds'];
+	const invalidDurations = ['minutes', 'NaN', 'abc', ''];
```

Comment:
Can we include the items from the previous array? I would expect that they are still invalid.
```suggestion
	const invalidDurations = ['20h', '20 hours', '20s', '20 seconds','minutes', 'NaN', 'abc', ''];
```